pom.xml

```diff
@@ -11,7 +11,7 @@
     <version>1.0.11-SNAPSHOT</version>
     <packaging>hpi</packaging>
     <properties>
-        <jenkins.version>2.37</jenkins.version>
+        <jenkins.version>2.7.3</jenkins.version>
```

Comment:
That was wrong version..
I told you it needs 2.37 which is way younger then 2.7.3 :)

Reply:
Ok, but it breaks when I use 2.37. Are you sure it uses 2.7.3? Can we verify this again?

Reply:
I don't see why we can't use 2.7.3 since it's working for me. Correct me if I'm wrong. :)

Reply:
Is there any reason I can't use the latest release of Jenkins (2.150.3)?
https://jenkins.io/doc/upgrade-guide/
doesn't seem to have 2.37

Reply:
Proposal: we support 12 months of LTS releases. For this change, 2.107.1 would be the minimum version since it's the oldest LTS less than 12 months old.

@craigatgoogle this could apply to the GKE plugin as well.

Reply:
SGTM